applications/java/deployment/java-ingress.yaml

```diff
@@ -0,0 +1,21 @@
+apiVersion: networking.k8s.io/v1
+kind: Ingress
+metadata:
+  annotations:
+    nginx.org/mergeable-ingress-type: minion
+    nginx.org/rewrites: serviceName=java-appmod-service rewrite=/
+  name: java-appmod-service-ingress
+  namespace: appmod-java
+spec:
+  ingressClassName: nginx
+  rules:
+  - host: '*.amazonaws.com'
+    http:
+      paths:
+      - backend:
+          service:
+            name: java-appmod-service
+            port:
+              number: 8080
+        path: /
```

Comment:
This won't work with Argo ingress that we install later - can we change the path to /java-app?

Reply:
Updated